### PR TITLE
Added links to fraternity and committee page

### LINKED ISSUE #635: Clickable names of committees / fraternities on pages of members

### PR DESCRIPTION
**Pull request by gafderks**
Fixes #635

### CODE REVIEW

**Comment by jszanto** [comment]
Wouldn't it be better to just modify this code https://github.com/GEWIS/gewisweb/blob/master/module/Decision/src/Decision/Service/Member.php#L53 so it includes the organ instead of only the abbreviation?

**Comment by gafderks** [comment]
@jszanto How do you suggest I change the array like `Array ( [BAC] => Array ( [0] => Voorzitter ) )` to include the organ?

**Comment by jszanto** [comment]
not a 100% sure if this will work, but you could try `organ => functions`?

**Comment by gafderks** [comment]
So use an object as a key in the array?

**Comment by jszanto** [comment]
Yes

**Comment by gafderks** [comment]
From the [docs](https://secure.php.net/manual/en/language.types.array.php):
> Arrays and objects can not be used as keys. Doing so will result in a warning: Illegal offset type.

**Comment by jszanto** [comment]
Okay, how about
```
BAC => ['organ' => object, 'functions' => functions],..
```

File: module/Decision/src/Decision/View/Helper/Organ.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
```
Comment (jszanto) [inline module/Decision/src/Decision/View/Helper/Organ.php]: Is this file still needed?

Comment (kokx) [inline module/Decision/src/Decision/View/Helper/Organ.php]: I can't find any occurence of the `findOrgan` in the code. So I would guess not.

File: module/Decision/view/decision/member/view.phtml [hunk 2]
```diff
@@ -44,8 +44,15 @@ use Decision\Model\Address;
...
-                    <li><?= $organ ?><?php if (!empty($functions)): ?> (<?= implode(', ', $functions) ?>)<?php endif ?></li>
+                <?php foreach ($memberships as $abbr => $install): ?>
+                    <li><a href="<?= $this->url('home/organ', ['type' => $install['organ']->getType(), 'abbr' =>
+                            $abbr ]) ?>"><?= $abbr ?></a>
+                        <?php $functions = $install['functions']; ?>
+                        <?php if (!empty($functions)): ?> (<?=
+                            $functions ?>)
+                        <?php
+                        endif
+                        ?></li>
```
Comment (kokx) [inline module/Decision/view/decision/member/view.phtml]: These three lines should be on one line.

File: module/Decision/view/decision/organ/show.phtml [hunk 3]
```diff
@@ -8,7 +8,10 @@ $this->headTitle($organ->getName()); ?>
             <h2><?= $this->translate('Current members') ?></h2>
             <ul>
                 <?php foreach ($currentMembers as $membership): ?>
-                    <li><?= $membership['member']->getFullName() ?> <?php if (!empty($membership['functions'])): ?> (<?= implode(', ', $membership['functions']) ?>)<?php endif ?></li>
+                    <li><a href="<?= $this->url('member/view', ['lidnr' => $membership['member']->getLidnr()])
+                        ?>"><?= $membership['member']->getFullName() ?></a> <?php if (!empty
+                        ($membership['functions'])):
+                            ?> (<?= implode(', ', $membership['functions']) ?>)<?php endif ?></li>
```
Comment (kokx) [inline module/Decision/view/decision/organ/show.phtml]: The indendation here looks off.

File: module/Decision/src/Decision/View/Helper/Organ.php (hunk 1 above)
Comment (gafderks) [inline module/Decision/src/Decision/View/Helper/Organ.php]: I have addressed these issues.

File: module/Decision/config/module.config.php [hunk 4]
```diff
@@ -329,5 +329,10 @@
                 ]
             ]
         ]
-    ]
+    ],
+    'view_helpers' => [
+        'invokables' => [
+            'findOrgan' => 'Decision\View\Helper\Organ'
+        ],
+    ],
```
Comment (jszanto) [inline module/Decision/config/module.config.php]: I think this should be removed too

Comment (gafderks) [inline module/Decision/config/module.config.php]: True
